src/InitOMPasses.hpp

```diff
@@ -5,6 +5,10 @@
 #include "mlir/Pass/Pass.h"
 #include "src/Pass/Passes.hpp"
 
+#ifdef __NNPA__
```

Comment:
This header is missing the copyright notice. Can you add it pls ?

Reply:
Done. We should have a separate PR for this, I see some other files missing the copyright.